app/javascript/react/components/ReviewTile.test.js

```diff
@@ -0,0 +1,49 @@
+import React from "react"
+import Enzyme, { mount } from "enzyme"
+
+import ReviewTile from './ReviewTile'
+
+describe("ReviewTile", () => {
+  let wrapper
+  let wrapper2
+
+  beforeEach(() => {
+    wrapper = mount(
+      <ReviewTile
+        id="3"
+        userId="3"
+        rating="5"
+        body="Great Wifi"
+        speedData=""
+      />
+    )
+    wrapper2 = mount(
+      <ReviewTile
+        id="4"
+        userId="3"
+        rating="3"
+        body=""
+        speedData="3000.9"
+      />
+    )
+  })
```

Comment:
I like the forethought here, however ideally we add another nested `describe` block here, such that you have two nested`describe` blocks that evaluate the two different review tiles. 

Otherwise, you are mounting two review tiles for each test, but only ever using one.